embedded-graphics/src/primitives/line.rs

```diff
@@ -65,76 +60,318 @@ impl Dimensions for Line {
 impl Line {
     /// Create a new line
     pub const fn new(start: Point, end: Point) -> Self {
-        Line { start, end }
+        Self { start, end }
+    }
+}
+
+/// Which side of the center line to draw on
+///
+/// Imagine standing on `start`, looking ahead to where `end` is. `Left` is to your left, `Right` to
+/// your right.
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+pub enum Side {
+    /// DOCS LOL
```

Comment:
This might not be the best doc comment :wink:. If `Side` isn't `pub` anymore these comments could be removed.